app/Services/FacultyService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\Models\Faculty;
+use App\Repositories\FacultyRepository;
+
+class FacultyService{
+
+    const NOT_FOUND_MESSAGE = "Faculty not found";
+
+    private readonly FacultyRepository $facultyRepository;
+    
+    public function __construct(FacultyRepository $facultyRepositoryInstance)
+    {
+        $this->facultyRepository = $facultyRepositoryInstance;
+    } 
```

Comment:
Fix this injecting with PHP-8.1 Inject way, 
I've commented at first or Check DonationService